web/yo/app/scripts/controllers/VariantCtrl.js

```diff
@@ -430,13 +428,13 @@ angular.module('oncokbApp')
             $scope.$watch('view.selectedCancerType', function(n) {
                 if (n) {
                     if ($scope.view.selectedSubtype) {
-                        var mappedSubtypes = findSubtypesByCancerType(n.cancerType);
+                        var mappedSubtypes = findSubtypesByCancerType(n.mainType.name);
                         if (!containSubtype($scope.view.selectedSubtype, mappedSubtypes)) {
                             $scope.view.filteredSubtypes = mappedSubtypes;
                             $scope.view.selectedSubtype = '';
                         }
                     } else {
-                        $scope.view.filteredSubtypes = findSubtypesByCancerType(n.cancerType);
+                        $scope.view.filteredSubtypes = findSubtypesByCancerType(n.mainTypeName);
```

Comment:
@JiaoJiao123 thanks! I updated the PR. If you think it's good, could you merge it?